src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/CourtEvent.java

```diff
@@ -77,15 +77,15 @@
     @Column(name = "START_TIME", nullable = false)
     private LocalDateTime startTime;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.EAGER)
     @NotFound(action = IGNORE)
```

Comment:
I thought having the NotFound meant it would be EAGER loaded anyway?

Reply:
> Yes so being explicit as it generates a warning